Syntaxes/Julia.tmLanguage

```diff
@@ -244,13 +244,13 @@
 			<array>
 				<dict>
 					<key>match</key>
-					<string>\b(?:function|stagedfunction|type|immutable|macro|quote|abstract|bitstype|typealias|module|baremodule|new)\b</string>
```

Comment:
huh? it's not a keyword outside of type context though, and it looks very very wrong whenever it's highlighted as a keyword but used as a variable name outside of type context - most code isn't in type context